webrender/src/frame_builder.rs

```diff
@@ -951,6 +951,7 @@ impl FrameBuilder {
                 }
             }
             BoxShadowKind::Shadow(rects) => {
+                assert!(blur_radius > 0.0);
```

Comment:
It breaks down into a simple rect below - https://github.com/servo/webrender/blob/master/webrender/src/frame_builder.rs#L910

We can probably merge the two cases. I'll do that in a follow up. I'm just going through the gecko reftests.